lib/acts_as_list/active_record/acts/position_column_method_definer.rb

```diff
@@ -49,6 +52,38 @@ def self.define_class_methods(caller_class, position_column, touch_on_update)
 
       private
 
+      # Update recordset to start at the given position
+      define_singleton_method :shuffle_to do |position|
```

Comment:
Help me understand, why is this needed?

Reply:
@swanandp Currently if you have a unique index on `position_column`, reordering a list is *slow* - one query per item in the list, to avoid collisions. 

`shuffle_to` uses the same number of queries regardless of list size, and doesn't generate collisions if you have a unique index. 

If you have a few hundred items in a list, this is much, much faster.